core/labeled-replies-3.3.md

```diff
@@ -40,7 +40,7 @@ Additionally, labeled responses allow bouncers with multiple connected clients t
 
 ### Dependencies
 
-This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and relies on support for the [`batch`](/specs/extensions/batch-3.2.html) capability which MUST be negotiated at the same time.
+This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and uses the [`batch`](/specs/extensions/batch-3.2.html) capability which SHOULD be negotiated at the same time.
```

Comment:
This specification should be independent of the use of `batch` - if needed, they can be used in tandem and negotiated at the same time. "_... and, if needed, the batch capability should be negotiated at the same time_" 

The `draft/label` tag can be useful in the absence of `batch`. While batches are useful and can aid in keeping relevant content together, it should not be required for all uses of the `draft/labeled-response` message tag. There are instances where it may be necessary to uniquely identify a labeled response for content that does not belong to a batch, such as in ircv3/ircv3-ideas#6.

IRC Discussion
```
<prawnsalad> ideally, for any command created, it should be i send command X, and i expect a response for command X. error, ack, or response. so the client isnt sat there waiting for something that may or may not actually happen
<MuffinMedic> so, response is the batch, error would be thins like NO_CHATHISTORY_FOUND, ACCESS_DENIED, etc....?
<prawnsalad> "CHATHISTORY ERR_NOTFOUND/ERR_NOACCESS" would be my on-a-whim suggestion
<dx> space after ERR
<prawnsalad> "CHATHISTORY ERR <err name>" dx, yea this would be better
<MuffinMedic> that prevents a new issue though
<MuffinMedic> what if they request multiple chat histories? it would need to include a random ID parameter in the chathistory command that the server returns with the error
<dx> MuffinMedic: the labeled replies spec can take care of that
<MuffinMedic> dx: so send "@label=ID SCROLLBACK target.." then return "@label=ID SCROLLBACK ERR msg"?
<dx> yeah
```